includes/emails/class-wcs-email-customer-notification.php

```diff
@@ -80,15 +89,42 @@ public function trigger( $subscription_id ) {
 			|| ! WC_Subscriptions_Email_Notifications::should_send_notification()
 			|| WCS_Action_Scheduler_Customer_Notifications::is_subscription_period_too_short( $subscription )
 		) {
-			// TODO: add admin notice here if in admin
+			// Add order/subscription note.
+			if ( is_admin() ) {
```

Comment:
What are your thoughts on removing this `is_admin()` check?

I was thinking it would be helpful for debugging purposes and for merchants to know whenever a reminder email was scheduled but not sent.

Is the idea that if we ended up inside this condition, the scheduled action should've been deleted but wasn't? And so we want to silently not send the email unless it was requested by the merchant?

Reply:
Initially, I thought it would make sense to show the notice/info only for manual triggers, but now that you've raised it, perhaps it makes sense also for scheduled notifications. 

There might be some cases when this will generate false positive messages. If the notifications get disabled globally, there might be some notifications that will run until their respective scheduled actions are unscheduled, but it's fairly improbable as we mass-unschedule based on the AS group.

tl;dr is: I agree, let's drop the `is_admin()` check.

Reply:
Dropped it in [f7d14a2](https://github.com/Automattic/woocommerce-subscriptions-core/pull/710/commits/f7d14a2a8d83c5ef8963721cf0716f0dcef0473c).